worklets/Overview.bs

```diff
@@ -285,85 +303,130 @@ The <a>module responses map</a> exists to ensure that {{WorkletGlobalScope}}s cr
 times contain the same set of script source text and have the same behaviour. The creation of
 additional {{WorkletGlobalScope}}s should be transparent to the author.
 
-Note:
+<div class='note'>
     Practically user-agents aren't expected to implement the following algorithm using a
-    thread-safe map. Instead when {{Worklet/import}} is called user-agents can fetch the module
+    thread-safe map. Instead when {{Worklet/import()}} is called user-agents can fetch the module
     graph on the main thread, and send the fetched sources (the data contained in the <a>module
     responses map</a>) to each thread which has a {{WorkletGlobalScope}}.
 
     If the user agent wishes to create a new {{WorkletGlobalScope}} it can simply sent the list of
     all fetched sources from the main thread to the thread which owns the {{WorkletGlobalScope}}.
+</div>
 
 When the <dfn method for=Worklet>import(|moduleURL|)</dfn> method is called on a {{Worklet}} object,
 the user agent <em>must</em> run the following steps:
     1. Let |promise| be <a>a new promise</a>.
 
-    2. Let |resolvedModuleURL| be the result of <a>parsing</a> the |moduleURL| argument relative to
+    2. Let |worklet| be the current {{Worklet}}.
+
+    3. Let |resolvedModuleURL| be the result of <a>parsing</a> the |moduleURL| argument relative to
         the <a>relevant settings object</a> of <b>this</b>.
 
-    3. If this fails, reject |promise| with a "<a>SyntaxError</a>" <a>DOMException</a> and abort
+    4. If this fails, reject |promise| with a "<a>SyntaxError</a>" <a>DOMException</a> and abort
         these steps.
 
-    4. Ensure that there is at least one {{WorkletGlobalScope}} in the <a>worklet's
-        WorkletGlobalScopes</a>. If not <a>create a WorkletGlobalScope</a> given the current
-        {{Worklet}}.
+    5. Return |promise|, and then continue running this algorithm <a>in parallel</a>.
+
+    6. Let |outsideSettings| be the <a>relevant settings object</a> of <b>this</b>.
+
+    7. Let |moduleResponsesMap| be |worklet|'s <a>module responses map</a>.
+
+    8. Let |workletGlobalScopeType| be |worklet|'s <a>worklet global scope type</a>.
+
+    9. Ensure that there is at least one {{WorkletGlobalScope}} in the <a>worklet's
+        WorkletGlobalScopes</a>.
```

Comment:
I think what you want here instead is a simple "If/Otherwise". Ensure might lead folks to think they can just make one up.